journal/models.py

```diff
@@ -54,10 +61,10 @@ class Meta:
         ]
 
     def __unicode__(self):
-        return "%s - %s" % (self.issnl, self.title) or ""
+        return self.title
 
     def __str__(self):
-        return "%s - %s" % (self.issnl, self.title) or ""
+        return self.title
```

Comment:
Aqui não poderá ocorrer um erro se title for nulo?
"TypeError: __str__ returned non-string (type NoneType)"